src/line.rs

```diff
@@ -0,0 +1,76 @@
+/// A macro for creating a [`Line`] using vec! syntax.
+///
+/// `line!` is similar to the [`vec!`] macro, but it returns a [`Line`] instead of a `Vec`.
+///
+/// # Examples
+///
+/// * Create a [`Line`] containing a vector of [`Span`]s:
+///
+/// ```rust
+/// # use ratatui::prelude::*;
+/// use ratatui_macros::line;
+///
+/// let line = line!["hello", "world"];
+/// let line = line!["hello".red(), "world".red().bold()];
+/// ```
+///
+/// * Create a [`Line`] from a given [`Span`] and size:
```

Comment:
comment from other PR
```suggestion
/// * Create a [`Line`] that repeats a [`Span`] a given amount of times:
```